grouper/model_soup.py

```diff
@@ -861,6 +860,10 @@ def apply_changes(self, request):
             else:
                 setattr(self, key, value)
 
+    def apply_changes(self, request):
+        changes = json.loads(request.changes)
```

Comment:
This seems like something that doesn't belong in a model -- deserialisation is best handled at the view / validation / whatever layer.